terraform/aks/workspace_variables/sandbox_aks.tfvars.json

```diff
@@ -0,0 +1,20 @@
+{
+  "paas_app_environment": "sandbox",
+  "azure_resource_prefix": "s189p01-att",
+  "key_vault_resource_group": "s189p01-att-sbx-rg",
+  "key_vault_name": "s189p01-att-sbx-kv",
+  "key_vault_app_secret_name": "APPLY-APP-SECRETS-SANDBOX",
+  "key_vault_infra_secret_name": "BAT-INFRA-SECRETS-SANDBOX",
+  "cluster": "production",
+  "namespace": "bat-production",
+  "gov_uk_host_names": ["sandbox2.apply-for-teacher-training.service.gov.uk","sandbox2.apply-for-teacher-training.education.gov.uk"],
+  "webapp_memory_max": "1024Mi",
+  "worker_memory_max": "1024Mi",
+  "secondary_worker_memory_max": "1024Mi",
+  "clock_worker_memory_max": "1024Mi",
+  "webapp_replicas": 4,
+  "worker_replicas": 1,
+  "secondary_worker_replicas": 1,
+  "clock_worker_replicas": 0,
+  "postgres_flexible_server_sku": "GP_Standard_D2ds_v4"
```

Comment:
paas sandbox postgres is ha, so are we enabling for aks sandbox? via postgres_enable_high_availability?